### ISSUE TITLE
how to update

### ISSUE THREAD

**Issue by avichou** [BigBerny/magicquit#4]
how do we trigger update or even just get a notification/rss feed to be alerted of new releases ?

i've seen there was an update using macupdater, otherwise i would have never known about it, and wouldnt have gone on the official site to see that the current downloadable version is 1.4 whilst installed is 1.3 ...

**Comment by yeahitsjan** [comment]
I would second this. Atleast some messagebox that shows there is a new version available on the website would be enough.

Also, is there any way to donate something? MagicQuit is one of the first apps I install on any Mac in my family & myself.

**Comment by BigBerny** [comment]
Yeah, I know it's not ideal. As written [here](https://github.com/BigBerny/magicquit/issues/3#issuecomment-1878614499) I just don't have time at the moment to add a feature like this. But I'll keep it in mind.
